api/resource/factory.go

```diff
@@ -42,7 +42,11 @@ func (rf *Factory) Hasher() ifc.KustHasher {
 
 // FromMap returns a new instance of Resource.
 func (rf *Factory) FromMap(m map[string]interface{}) *Resource {
-	return rf.FromMapAndOption(m, nil)
+	res, err := rf.FromMapAndOption(m, nil)
+	if err != nil {
+		log.Fatal("Option must not be null")
```

Comment:
It looks like this line abandoned a non-nil error.
Could you make a change to print the substance of the returned error?